src/Fixer/ControlStructure/IncludeFixer.php

```diff
@@ -80,20 +82,12 @@ private function clearIncludies(Tokens $tokens, array $includies)
                     $this->removeWhitespaceAroundIfPossible($tokens, $braces['close']);
                     $tokens->clearTokenAndMergeSurroundingWhitespace($braces['open']);
                     $tokens->clearTokenAndMergeSurroundingWhitespace($braces['close']);
-
-                    $nextSiblingIndex = $tokens->getNonEmptySibling($includy['begin'], 1);
-                    if (!$tokens[$nextSiblingIndex]->isWhitespace()) {
-                        $tokens->insertAt($nextSiblingIndex, new Token(array(T_WHITESPACE, ' ')));
+                    if (!$tokens[$nextIndex]->isWhitespace()) {
```

Comment:
without this statement the test pass as well, which seems OK because line 91 repeats the same check and performs the same insert of the whitespace, is this something that could simplified?